### DISCUSSION TITLE
Net Neutrality and QoD

### DISCUSSION THREAD

**Discussion by veeratk** [camaraproject/QualityOnDemand#250] [General]
As you are aware many countries's telecom regulatory authority restricts "Discriminatory Treatment" to be provided by a service provider to a subscriber (exempting specialized services).

How we are countering this? Or countries that are enforcing net neutrality, QoD cannot be used.

Experts, please let me know your view on the same.

**Comment by veeratk** [comment]
@hdamker could you please provide guidance or views about Net Neutrality and QoD?

**Comment by hdamker** [comment]
Closing this discussion as we can't discuss local regulation topics within CAMARA.
